docs/development/howtocontribute.md

```diff
@@ -113,6 +113,23 @@ cd <zeppelin_home>/zeppelin-interpreter/src/main/thrift
 ./genthrift.sh
 ```
 
+### Run Selenium test
+
+Zeppelin has [set of integration tests](https://github.com/apache/zeppelin/tree/master/zeppelin-server/src/test/java/org/apache/zeppelin/integration) using Selenium. To run these test, first build and run Zeppelin and make sure Zeppelin is running on port 8080. Then you can run test using following command
+
+```
+TEST_SELENIUM=true mvn test -Dtest=[TEST_NAME] -DfailIfNoTests=false -pl 'zeppelin-interpreter,zeppelin-zengine,zeppelin-server'
+```
+
+For example, to run [ParagraphActionIT](https://github.com/apache/zeppelin/blob/master/zeppelin-server/src/test/java/org/apache/zeppelin/integration/ParagraphActionsIT.java),
+
+```
+TEST_SELENIUM=true mvn test -Dtest=ParagraphActionsIT -DfailIfNoTests=false -pl 'zeppelin-interpreter,zeppelin-zengine,zeppelin-server'
+```
+
+You'll need Firefox web browser installed in your development environment. While CI server uses 'Firefox 31.0' to run selenium test, it is good idea to install the same version.
```

Comment:
any instructions on how to get the exact version of Firefox? Firefox updates itself I think and it might be tricky to get/stay with the same version